cloudca/resource_cloudca_load_balancer_rule.go

```diff
@@ -118,7 +118,7 @@ func createLbr(d *schema.ResourceData, meta interface{}) error {
 
 	if instanceIdsPresent {
 		var instanceIds []string
-		for _, id := range d.Get("instance_ids").([]interface{}) {
+		for _, id := range d.Get("instance_ids").(*schema.Set).List() {
```

Comment:
The branch was out of date. It doesn't appear anymore when targeting the dev branch.